pageserver/src/layered_repository.rs

```diff
@@ -1353,12 +1353,10 @@ impl LayeredTimeline {
                 } else {
                     result.ondisk_nonrelfiles_dropped += 1;
                 }
+            } else if doomed_layer.get_seg_tag().rel.is_relation() {
```

Comment:
I don't think this is an improvement, I'd like to keep the if-branches symmetrical.

Maybe there's a better way to refactor this though. This is a recurring pattern in the function, checking `rel.is_relation()` and incrementing either `relfiles_something` or `nonrelfiles_something`.

Reply:
Doesn't look better to me neither, I've spent a few minutes on that, and decided to apply the suggestion. I see the following option:
```rust
match (doomed_layer.is_dropped(), doomed_layer.get_seg_tag().rel.is_relation()) {
    (true, true) => result.ondisk_relfiles_dropped += 1,
    (true, false) => result.ondisk_nonrelfiles_dropped += 1,
    (false, true) => result.ondisk_relfiles_removed += 1,
    (false, false) => result.ondisk_nonrelfiles_removed += 1,
}
```
what do you think about it?

Reply:
Works for me